HelpMyStreetFE/HelpMyStreetFE/Helpers/Extensions.cs

```diff
@@ -7,6 +8,62 @@
 
 namespace HelpMyStreetFE
 {
+    public static class SupportActivityHelpers
+    {
+        public static string FriendlyName(this SupportActivities activity)
+        {
+            return activity switch
+            {
+                SupportActivities.Shopping => "Shopping",
+                SupportActivities.FaceMask => "Face Covering",
+                SupportActivities.CheckingIn => "Check in",
+                SupportActivities.CollectingPrescriptions => "Prescriptions",
+                SupportActivities.Errands => "Errands",
+                SupportActivities.DogWalking => "Dog Walking",
+                SupportActivities.MealPreparation => "A Meal",
+                SupportActivities.PhoneCalls_Friendly => "Friendly chat",
+                SupportActivities.PhoneCalls_Anxious => "Supportive chat",
+                SupportActivities.HomeworkSupport => "Homework",
+                _ => "Other"
+            };
+        }
+
+        public static string Icon(this SupportActivities activity)
+        {
+            return activity switch
+            {
+                SupportActivities.Shopping => "shopping.svg",
+                SupportActivities.FaceMask => "face-covering.svg",
+                SupportActivities.CheckingIn => "check-in.svg",
+                SupportActivities.CollectingPrescriptions => "prescriptions.svg",
+                SupportActivities.Errands => "Errands.svg",
+                SupportActivities.DogWalking => "dog-walking.svg",
+                SupportActivities.MealPreparation => "meal.svg",
+                SupportActivities.PhoneCalls_Friendly => "friendlychat.svg",
+                SupportActivities.PhoneCalls_Anxious => "supportivechat.svg",
+                SupportActivities.HomeworkSupport => "homework.svg",
+                _ => "question-mark.svg"
+            };
+        }
+
+        public static string Class(this SupportActivities activity)
+        {
+            return activity switch
+            {
+                SupportActivities.Shopping => "blue",
+                SupportActivities.FaceMask => "face-covering.svg",
```

Comment:
is this supposed to be "face-covering.svg"

Reply:
top spot!